2200002/SWEA_5431.py

```diff
@@ -0,0 +1,15 @@
+
+
+
+T = int(input())
+
+for i in range(1, i + 1):
+    N, K = map(int, input().split())
+    data = lsit(map(int, input().split()))
```

Comment:
깔끔합니다 👍 
```suggestion
    data = list(map(int, input().split()))
```
오타가 있는 것 같아요!